src/mame/drivers/p2000t.cpp

```diff
@@ -16,16 +16,58 @@ Philips P2000 1 Memory map
     Ports:
         00-09       Keyboard input
         10-1f       Output ports
-        20-2f       Input ports
+        20-2f       Input ports  (2c Hires reset port)
         30-3f       Scroll reg (T ver)
         50-5f       Beeper
+		68-6b       Hires communication ports
+					68: PIO A DATA status channel 
+					6a: PIO A ctrl status channel 
+					69: PIO B DATA status channel 
+					6b: PIO B ctrl status channel 
         70-7f       DISAS (M ver)
-        88-8B       CTC
+		88-8B       CTC
         8C-90       Floppy ctrl
         94          RAM Bank select
 
+
     Display: SAA5050
 
+2020-12-16 Bekkie: 
+		- Added 80/40 character per line switch support [out 0,0 / out 0,1]
+		  Required an additional PCB in the passed
+		- Added Hires graphics card support
+          Additional rom "GOS36.bin" and Basic loader "Taalje 1.1 32k.cas" is required
+		  see: P2000T preservation page [https://github.com/p2000t/software]
+		- TODO: Adding m2200 floppy drive support
+		- TODO: Adding MSX mouse emulator support
```

Comment:
Please don’t put change logs in header comments.  We have version control for a reason.